improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -494,5 +494,36 @@ def test_dimension_preservation(self):
         self.assertListEqual(result_coord_order, expected_coord_order)
 
 
+class Test_alinfit(IrisTest):
+    """Test the alinfit method"""
+
+    def setUp(self):
+        """Creates some random data to represent X and Y."""
+        array_size = 25
+        self.X = np.random.random(array_size)
+        self.Y = np.random.random(array_size)
+
+    def use_lstsq(self):
+        """Uses numpy's leastsquare algorithm to fit the data as a comparison"""
+        X = np.stack([self.X, np.ones(len(self.X))]).T
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "X" doesn't conform to snake_case naming style](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)